primefaces/src/main/java/org/primefaces/component/fileupload/FileUploadRenderer.java

```diff
@@ -38,27 +38,24 @@
 import javax.faces.convert.ConverterException;
 import java.io.IOException;
 import java.util.Arrays;
-import java.util.logging.Logger;
 
 public class FileUploadRenderer extends CoreRenderer {
 
-    private static final Logger LOGGER = Logger.getLogger(FileUploadRenderer.class.getName());
-
     @Override
     public void decode(FacesContext context, UIComponent component) {
         FileUpload fileUpload = (FileUpload) component;
-        if (!fileUpload.isDisabled()) {
-            if (!context.getExternalContext().getRequestContentType().toLowerCase().startsWith("multipart/")) {
-                LOGGER.severe(() -> "Decoding FileUpload requires contentType \"multipart/form-data\"." +
-                        " Skipping clientId: " + component.getClientId(context));
-                return;
-            }
+        if (fileUpload.isDisabled()) {
+            return;
+        }
+        if (!context.getExternalContext().getRequestContentType().toLowerCase().startsWith("multipart/")) {
+            logDevelopmentWarning(context,
+                    "Decoding FileUpload requires contentType \"multipart/form-data\". Skipping clientId: " + component.getClientId(context));
```

Comment:
remove the word "Skipping", we dont skip anymore

Reply:
Removed skipping!  new message is `"Decoding FileUpload requires contentType \"multipart/form-data\" for clientId: " + component.getClientId(context)`